example/example-Tzip12BigMapOffChain.ts

```diff
@@ -0,0 +1,133 @@
+import { CONFIGS } from '../integration-tests/config';
+import { MichelsonMap, TezosToolkit } from '@taquito/taquito';
+import { importKey } from '@taquito/signer';
+import { fa2ForTokenMetadataView } from '../integration-tests/data/fa2-for-token-metadata-view';
+import { b58cencode, char2Bytes, Prefix, prefix } from '@taquito/utils';
```

Comment:
@taquito/utils should be added to the package.json file

Reply:
added